ui/lib/kv/addon/routes/secret/details/index.js

```diff
@@ -5,17 +5,35 @@
 
 import Route from '@ember/routing/route';
 import { breadcrumbsForSecret } from 'kv/utils/kv-breadcrumbs';
+import { inject as service } from '@ember/service';
 
 export default class KvSecretDetailsIndexRoute extends Route {
-  setupController(controller, resolvedModel) {
+  @service store;
+
+  // polled by controller and called once when route initializes
+  async fetchSyncStatus(model) {
+    const { backend: mount, path: secretName } = model;
+    const syncAdapter = this.store.adapterFor('sync/association');
+    return syncAdapter.fetchDestinations({ mount, secretName });
+  }
```

Comment:
Is there any reason this can't live in the secret details page component? It would make for easier testing.

Reply:
I initially wanted it to live there - but just ended up copying the existing pattern used for polling the pki tidy status. 

I'll move it to the component if we both agree it'd be better there!